src/Member.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;
+
+public class Member {
+    private String fullName;
+
+    private static ArrayList indexesofActiveMembers = new ArrayList();
+
+    public static ArrayList getIndexesofActiveMembers() {
+        return indexesofActiveMembers;
+    }
+
+    public static void setMemberOffline(int i) {
+        Object object = indexesofActiveMembers.get(i);
+        indexesofActiveMembers.remove(object);
```

Comment:
not sure if I understand why you just not doing indexesofActiveMembers.remove(i);, as you put indexes in activeIndexes list, if you define list using generics there will be no issue with it